### PR TITLE
Clean up README content.

### PR DESCRIPTION
**Pull request by barelyknown**
I edited the README when I re-read it this morning.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,20 +187,17 @@ The [JSON API] spec defines relationships with `links` objects the make each
 JSON document discoverable, providing URLs for `related` and `self`.
 
 This implementation takes the posture that your application's resources
-to not need a complex abstraction but a simple implemenation of a solid
-specification. So not only is this project great for getting started
+do not need a complex abstraction but a simple implemenation of a solid
+specification. So this project great for getting started
```
Comment (pixelhandler) [inline README.md]: s/So this project great/So this project is great/

**Comment by barelyknown** [comment]
Typo in the copy edit. Lovely!

**Comment by pixelhandler** [comment]
@barelyknown that is awesome that you proofread and corrected the typos in the Readme file, and grammar too :)

The Readme was mostly a rant I pounded out on the keyboard yesterday. It's nice to have these corrections, thanks!

**Comment by barelyknown** [comment]
:+1:

**Pull request merged**